### PR TITLE
(GH-29) Don't use Installation Arguments Requirement

### PR DESCRIPTION
**Pull request by gep13**
- Close GH-29

### CODE REVIEW

File: src/chocolatey.package.validator/infrastructure.app/rules/InstallScriptsShouldntUseInstallArgumentsRequirement.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            foreach (var packageFile in files)
+            {
+                string extension = Path.GetExtension(packageFile.Path).to_lower();
+                if (extension != ".ps1" && extension != ".psm1") continue;
+
+                var contents = packageFile.GetStream().ReadToEnd().to_lower();
+
+                if (contents.Contains("installArguments") ||
+                    contents.Contains("installerArguments") ||
+                    contents.Contains("chocolateyInstallArguments"))
```
Comment (ferventcoder) [inline src/chocolatey.package.validator/infrastructure.app/rules/InstallScriptsShouldntUseInstallArgumentsRequirement.cs]: After the stream is to_lower, I'm not sure any of these are going to match. ;)

Comment (gep13) [inline src/chocolatey.package.validator/infrastructure.app/rules/InstallScriptsShouldntUseInstallArgumentsRequirement.cs]: doh! Good point!

Are they the correct environment variable names though?

Comment (ferventcoder) [inline src/chocolatey.package.validator/infrastructure.app/rules/InstallScriptsShouldntUseInstallArgumentsRequirement.cs]: :+1:

**Comment by gep13** [comment]
@ferventcoder rebased and pushed.

**Pull request merged**